static/app/views/organizationGroupDetails/groupEvents.tsx

```diff
@@ -141,21 +179,32 @@ class GroupEvents extends Component<Props, State> {
   }
 
   renderBody() {
+    const {issueCategory, id} = this.props.group;
+
     let body: React.ReactNode;
 
+    if (id && issueCategory === IssueCategory.PERFORMANCE) {
+      return this.renderPerfIssueEvents();
+    }
     if (this.state.loading) {
       body = <LoadingIndicator />;
     } else if (this.state.error) {
       body = <LoadingError message={this.state.error} onRetry={this.fetchData} />;
-    } else if (this.state.eventList.length > 0) {
+    }
+    // TODO - in the future we may share the perfIssueEvents with errorEvents
```

Comment:
I usually reserve TODO for tech debt or definite future changes that have to be made, if it's a 'maybe' we should just remove it and we'll see what the future has in store 😄

Reply:
removed!